### PR TITLE
Fixes after testing.

### PR DESCRIPTION
**Pull request by revit13**
This PR contains various fixes to README following testing.

Signed-off-by: Revital Sur <eres@il.ibm.com>
Co-authored-by: Doron Chen <cdoron@il.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ make docker-push
 
 ## Register as a Fybrik module
 
-To register HWM as a Fybrik module apply `hello-world-module.yaml` to the fybrik-system namespace of your cluster.
+To register HWM(Hello World Module) as a Fybrik module apply `hello-world-module.yaml` to the fybrik-system namespace of your cluster.
```
Comment (cdoron) [inline README.md]: please add space between "HWM" and "(Hello World Module)"

File: README.md [hunk 2]
```diff
@@ -159,22 +162,21 @@ Deploy `FybrikApplication` in `default` namespace:
 ```bash
 kubectl apply -f fybrikapplication.yaml -n default
 ```
-3.  Check if `FybrikApplication` successfully deployed:
+3.  Run the following command to wait until the `FybrikApplication` is ready:
```
Comment (cdoron) [inline README.md]: I suggest that you be more specific. For instance: wait until the Status of the `FybrikApplication` is 'ready'

Comment (revit13) [reply README.md]: Done thanks

File: README.md (hunk 1 above)
Comment (revit13) [reply README.md]: Done thanks

**Review by cdoron — APPROVED** [review]

**Review by roytman — APPROVED** [review]

**Pull request merged**